src/server/handle-api.js

```diff
@@ -15,26 +15,35 @@ export default ({ server, config }) => {
     path: '/api/v1/{query*}',
     handler: (req, reply) => {
 
-      const base = `${config.siteUrl}/wp-json/wp/v2`
+      const base = `${normalizePath(config.siteUrl)}/wp-json/wp/v2`
```

Comment:
true – up to you – it looks like you could use it as you're not dealing with protocols etc, just a straight path?